internal/libs/flowrate/flowrate.go

```diff
@@ -273,3 +273,17 @@ func (m *Monitor) waitNextSample(now time.Duration) time.Duration {
 	}
 	return now
 }
+
+// CurrentTransferRate returns the current transfer rate
+func (m *Monitor) CurrentTransferRate() int64 {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	_ = m.update(0)
```

Comment:
why are we updating/modifying the state of the monitor's average here?

Reply:
Removed.